### ISSUE TITLE
Python 2 EOL

### ISSUE THREAD

**Issue by louwers** [feinstaub/webextension_local_filesystem_links#160]
`local-link-messaging-host.py` assumes `#!/usr/bin/env python` is Python 2. However, since Python 2 is deprecated, many Linux distributions including Arch Linux don't link `python` to `python2` anymore. Some don't even ship with Python 2 at all anymore.

That's probably also why the native host doesn't work on my computer.

**Comment by AWolf81** [comment]
@louwers you're right, the native app should be updated to Python 3 because of EOL of Python 2. But I think Pyinstaller is bundling the executable of Python so it's available.
Not sure if that's also happening on Linux but for Windows, it will work if you're using the installer.

Anyway, if it's really because of Python. Could you please check the native host app in the following branch https://github.com/feinstaub/webextension_local_filesystem_links/tree/host-app-python3?

It should work with Python 2 & Python 3. But I haven't tested Python 2 yet. Do you think I have to test Python 2 or is it OK to switch to Py3?

Using the app should work by cloning and switching to the branch and running `./native-host/src/install_host.sh`.

The native app isn't using anything special so the switch to Python 3 wasn't that hard.

**Comment by louwers** [comment]
Sorry for the late response @AWolf81.
That branch **does** indeed work for me.

**Comment by sarahbesic** [comment]
I don't know if my topic sticks to this issue:

I used the macOS "native-app-bundle-mac". Everything nicely installed. Gatekeeper has been temporarily deactived via 
`sudo spctl --master-disable`.
Nevertheless clicking a file link within Firefox (with the Local Filesystem Links extension actived) results in a dialog:
**"Python" is damaged and can't be opened. You should move it to the Trash.**
System: macOS 10.15.6 Catalina
Python version installed detected via Terminal:
`python --version`
**Python 2.7.16**
Obviously the links are detected correctly but not processed properly because of an error in connection with the local link messeging host.
